prowler/providers/aws/services/mq/mq_service.py

```diff
@@ -39,3 +55,5 @@ class Broker(BaseModel):
     name: str
     id: str
     region: str
+    auto_minor_version_upgrade: bool = False
+    tags: Optional[list] = []
```

Comment:
If you are going to assign tags in the check, please ensure to attach it first in the describe call